java/client/build.gradle

```diff
@@ -184,7 +184,7 @@ publishing {
             groupId = 'io.valkey'
             artifactId = 'valkey-glide'
             // version is replaced during released workflow java-cd.yml
-            version = "255.255.255"
+            version = System.getenv("GLIDE_LOCAL_VERSION") ?: "255.255.255";
```

Comment:
@jonathanl-bq @Yury-Fridlyand let me know if you spot any conflicts with our build pipelines